js/private/js_binary.bzl

```diff
@@ -168,6 +168,19 @@ _ATTRS = {
         values = _LOG_LEVELS.keys(),
         default = "error",
     ),
+    "patch_node_fs": attr.bool(
```

Comment:
The child process patch will also have a flag? `patch_node_child_process`?

Reply:
I think the child process fs patches should just be implied if this is true since they are the same fs patches applied to spawn processes; just more a more robust implementation that covers the child spawn case when we get to it for 1.1

Reply:
Ah yeah, you're correct... it's the same patches but just properly applied to sub-processes as well